velox/exec/tests/AggregationFuzzer.cpp

```diff
@@ -414,6 +417,33 @@ CallableSignature AggregationFuzzer::pickSignature() {
   return signature;
 }
 
+std::vector<std::string> AggregationFuzzer::generateGroupingKeys(
+    std::vector<std::string>& names,
+    std::vector<TypePtr>& types) {
+  auto numGroupingKeys =
+      boost::random::uniform_int_distribution<uint32_t>(1, 5)(rng_);
+  std::vector<std::string> groupingKeys;
+  for (auto i = 0; i < numGroupingKeys; ++i) {
+    groupingKeys.push_back(fmt::format("g{}", i));
+
+    // Pick random scalar type.
+    types.push_back(vectorFuzzer_.randType(0 /*maxDepth*/));
+    names.push_back(groupingKeys.back());
+  }
+  return groupingKeys;
+}
+
+std::vector<RowVectorPtr> AggregationFuzzer::generateInputData(
+    std::vector<std::string> names,
+    std::vector<TypePtr> types) {
+  auto inputType = ROW(std::move(names), std::move(types));
+  std::vector<RowVectorPtr> input;
+  for (auto i = 0; i < 10; ++i) {
```

Comment:
Do we want to specify the number of rows in the vector as a parameter with default of 10?
```
int32_t batchSize = 10
```